cypress/features/regression/designSystem/menuDefaultDivider.feature

```diff
@@ -0,0 +1,15 @@
+Feature: Design Systems Menu component - divider story
+  I want to check Design Systems Menu component default divider story properties
+
+  Background: Open Design Systems Menu component default divider page
+    Given I open design systems default_divider "Menu" component in no iframe
+
+  @positive
+  Scenario: Check the persistence of Menu component
+    # When I open design systems default "Menu" component in no iframe
+    Then Menu elements are visible
+
+  @positive
+  Scenario: Check the size of the second expandable element of Menu
+    When I invoke third expandable Menu component
```

Comment:
I think that's better for the business reader
```suggestion
    When I hover over third expandable Menu component
```

Reply:
updated